unblob/plugins.py

```diff
@@ -79,3 +81,25 @@ def _import_modules(
             modules.append(mod)
 
         return modules
+
+    def add_handlers_to(
```

Comment:
The `add_handlers_to` name is a bit ambiguous for me as, maybe something like `extend_handlers_from_plugins` would describe this better?